xcaddy run with go 19 does not work

Hi,
when I try to run `xcaddy run -config foo.json` in the root directory of [caddy-l4](https://github.com/mholt/caddy-l4) I get the following error:
```
2022/09/09 23:50:15 [ERROR] exec: "caddy.exe": cannot run executable found relative to current directory
```
I found https://github.com/golang/go/issues/53962 and the workaround via setting the `GODEBUG=execerrdot=0` environment variable works.  
Can xcaddy fix this in a better way?

--- a/cmd/main.go
+++ b/cmd/main.go
@@ -164,10 +164,11 @@ func runBuild(ctx context.Context, args []string) error {
 }
 
 func getCaddyOutputFile() string {
+	f := "." + string(filepath.Separator) + "caddy"
 	if runtime.GOOS == "windows" {
-		return "caddy.exe"
+		f += ".exe"
 	}
-	return "." + string(filepath.Separator) + "caddy"
+	return f
 }
 
 func runDev(ctx context.Context, args []string) error {

